proxy/http/HttpConfig.h

```diff
@@ -670,6 +671,7 @@ struct OverridableHttpConfigParams {
   MgmtInt default_buffer_size_index;
   MgmtInt default_buffer_water_mark;
   MgmtInt slow_log_threshold;
+  MgmtInt ssl_client_verify_server;
```

Comment:
Besides, if this is now changing where this is more than 0|1, it needs to a) be documented, and b) Update the RecordsConfig.cc accordingly.